includes/Core/Assets/Assets.php

```diff
@@ -533,9 +550,10 @@ private function get_inline_base_data() {
 			'homeURL'          => home_url(),
 			'referenceSiteURL' => esc_url_raw( $site_url ),
 			'userIDHash'       => md5( $site_url . $current_user->ID ),
-			'adminRoot'        => esc_url_raw( get_admin_url() . 'admin.php' ),
-			'assetsRoot'       => esc_url_raw( $this->context->url( 'dist/assets/' ) ),
+			'adminURL'         => esc_url_raw( get_admin_url() . 'admin.php' ),
```

Comment:
The tests appear to expect this to result in a URL like `http://something.test/wp-admin` (why `admin.php` here?)

Also, what is/will this used for? It doesn't appear to be used anywhere.

Reply:
That is used by the datastore here.

Reply:
I'll update all these URLs to always include a trailing slash. It's mixed right now which may become problematic in the future.